src/main/java/com/educare/unitylend/controller/BorrowRequestController.java

```diff
@@ -138,6 +209,17 @@ else if(greaterThan){
      */
     @GetMapping("/get-all-borrow-requests")
     ResponseEntity<List<BorrowRequest>> getAllBorrowRequests() throws ControllerException {
-        return null;
+        try {
+            List<BorrowRequest> borrowRequestList = borrowRequestService.getAllBorrowRequests();
+            if (borrowRequestList == null || borrowRequestList.isEmpty()) {
+                log.error("No borrow requests created yet");
```

Comment:
If no borrow request are created, 
That is not an error, That is a warning.

And return the appropriate message in UI.